### DISCUSSION TITLE
Import/Export of database

### DISCUSSION THREAD

**Discussion by rad96** [bpepple/metron#287] [Q&A] [answered]
Is there an endpoint that will allow me to pull data from existing metron instance (like in json, files or something) and import it into, in example, my own instance, or to work with it as with GCD db

**Comment by bpepple** [comment, marked as answer]
I've thought about this and will most likely add a month dump of the db similar to GCD someday, but it's a ***really*** low priority item since the data is currently available to users through the API.
